src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -50,6 +51,7 @@
 
     private final ApplicationParams applicationParams;
     private final SecurityUtils securityUtils;
+    private static final  String repeatables = " because of ";
```

Comment:
Standard naming convention would make this `REPEATABLES`. However, since the value of this is just ` because of ` it probably makes more sense to change the way the strings are generated e.g. for the first one something like:
`String.format("Resource not found when getting case types for Jurisdiction: %s because of %s", jurisdictionId, e.getMessage())`